### DISCUSSION TITLE
Extender transcoding profiles

### DISCUSSION THREAD

**Discussion by trk2** [OpenSageTV/Community#42] [Developer discussions]
I've been relatively happy with the SageTVTranscoder plugin which has worked well to add basic .x265 playback capability to the HD300.  I've been playing with my own profiles for 4k to 1080 transcoding (and HDR tonemapping) which I can do offline through the plugin.  However, realtime transcoding has always been limited to 720x480 so I looked into the code to understand why.  

It seems miniplayer.java controls the logic of what profile to use for realtime transcoding.  As long as the bandwidth checks are passed, either the DVD or DVD6Ch profiles are used depending on the number of audio channels but both profiles use a 720x480 resolution.  It would be great if you could just change the DVD and DVD6Ch profiles in .properties but of course it get overwritten upon every restart.  

Basically this is the limit of my skills, but I was hoping if I pointed someone in the right direction we could figure out how to call custom profiles either through overwriting the DVD, DVD6Ch profile baked into Sage or by pointing to new profiles that are used instead.
